test/server/cards/01_SOR/units/HomeOneAllianceFlagship.spec.ts

```diff
@@ -0,0 +1,118 @@
+describe('Home One', function () {
+    integration(function (contextRef) {
+        describe('Home One\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['battlefield-marine'],
+                        spaceArena: ['home-one#alliance-flagship'],
+                        base: { card: 'echo-base', damage: 10 },
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors', 'cargo-juggernaut']
+                    }
+                });
+            });
+
+            it('should give Restore 1 to friendly units', function () {
+                const { context } = contextRef;
+
+                context.player1.clickCard(context.battlefieldMarine);
```

Comment:
Would also be good to test a friendly space unit and an enemy unit to confirm that the restore targeting is set up correctly